docker-compose-setup/init.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+cd "$(dirname "$0")/.." || exit
+mvn package -DskipTests
+docker run --rm -dit -v coatrack-admin:/home --name coatrack-admin ubuntu
```

Comment:
I am wondering if this stuff should go into our already existing docker build file and call that one from here? But maybe this should be discussed in one of the next calls.

Reply:
This part becomes redundant in #38 and will be removes in that PR. Maybe we can directly merge #38 to the CoatRack main to resolve this problem.